server/server_test.go

```diff
@@ -0,0 +1,44 @@
+package server
+
+import (
+	"github.com/gin-gonic/gin"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"net/http"
+	"net/http/httptest"
+)
+
+func performRequest(r http.Handler, method, path string) *httptest.ResponseRecorder {
+	req, _ := http.NewRequest(method, path, nil)
+	w := httptest.NewRecorder()
+	r.ServeHTTP(w, req)
+	return w
+}
+
+var _ = Describe("Server", func() {
+	var (
+		router   *gin.Engine
+		response *httptest.ResponseRecorder
+	)
+
+	BeforeEach(func() {
+		router = CreateRouter()
+	})
+
+	Describe("Version 1 API at /api/v1", func() {
+		Describe("The / endpoint", func() {
+			BeforeEach(func() {
+				response = performRequest(router, "GET", "/api/v1/")
+			})
+
+			It("Returns with Status 200", func() {
+				Expect(response.Code).To(Equal(200))
+			})
+
+			It("Returns the String 'Hello World'", func() {
```

Comment:
change the string to the new response. It is not 
"hello world" anymore

Reply:
Done. Implemented in commit 4753fcec18567d396c9b0d349c68fa68fc0e3767